tools/clang/unittests/HLSLExec/ShaderOpArith.xml

```diff
@@ -2552,11 +2552,11 @@
 
         void InitSharedMem(uint ix) {
           // Zero-init shared memory, with special cases
-          if (ix < 6)
+          if (ix < 7)
```

Comment:
This doesn't look right - `groupshared uint64_t g_uint64Share[6];` isn't large enough to index with a 7. Same for `xchgShare` below

Reply:
Oh, I might be confused, disregard this